helpdesk_mgmt_ticket_invoice/models/helpdesk_ticket.py

```diff
@@ -0,0 +1,10 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import fields, models
+
+
+class HelpdeskTicket(models.Model):
+    _inherit = "helpdesk.ticket"
+
+    ticket_invoice_id = fields.Many2one("account.move", relation="account_helpdesk")
```

Comment:
Field name:
```suggestion
    invoice_id = fields.Many2one("account.move", relation="account_helpdesk")
```

Same that other:  relation = model_1 + model_2 + rel:
```suggestion
    ticket_invoice_id = fields.Many2one("account.move", relation="account_move_helpdesk_ticket_rel")
```